server/index.js

```diff
@@ -7,6 +7,7 @@ const capitalize = require('to-case').capital;
 const pkg = require('../package.json');
 const { promisify } = require('util');
 const sequelize = require('sequelize');
+const umzug = require('./shared/database/umzug');
```

Comment:
⚠️ Loading this loads `./shared/database/index.js` too which in fact loads all our models before `sequelize.Promise` gets configured.